Tests/MockNetworkSession.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MockNetworkSession.swift
+//  NetworkingTests
+//
+//  Created by Michael Liberatore on 5/9/23.
+//  Copyright © 2023 Lickability. All rights reserved.
+//
+
+import Foundation
+import Networking
+
+/// A mocked version of `NetworkSession` to be used in tests. Allows specification of specific success or failure cases.
```

Comment:
```suggestion
/// A mocked version of `NetworkSession` to be used in tests. Allows specification of success or failure cases.
```

small thing, but I think "specification...of specific" is redundant and can be simplified